python/sglang/srt/entrypoints/openai/serving_base.py

```diff
@@ -62,6 +62,21 @@ async def handle_request(
             return self.create_error_response(
                 message=e.detail, err_type=str(e.status_code), status_code=e.status_code
             )
+        except ValueError as e:
+            # Check if this is a tool schema conflict error
```

Comment:
ValueError can be treated as 400 in general.

Let's remove the check and just do:

```
                return self.create_error_response(
                    message=str(e),
                    err_type="BadRequest",
                    status_code=400,
                )
```